### PR TITLE
Refactor theming of `Tabs`

### PR DESCRIPTION
**Pull request by NataliaVizintini**
Screenshot 2024-01-10 at 10 52 54

### CODE REVIEW

File: packages/admin/admin/src/tabs/Tabs.tsx [hunk 1]
```diff
@@ -82,24 +125,34 @@ function TabsComponent({
...
+                        </Content>
+                    );
                 } else if (child.props.forceRender) {
-                    return <div className={`${classes.content} ${classes.contentHidden}`}>{child.props.children}</div>;
+                    return (
+                        <Content ownerState={ownerState} {...slotProps?.content}>
+                            {child.props.children}
+                        </Content>
+                    );
                 }
```
Comment (jamesricky) [inline packages/admin/admin/src/tabs/Tabs.tsx:150]: This should be combined, as both conditions now return the same.

File: packages/admin/admin/src/tabs/Tabs.tsx [hunk 2]
```diff
@@ -82,24 +125,34 @@ function TabsComponent({
                     const { children, label, ...restTabProps } = child.props;
                     return <TabComponent label={label} {...restTabProps} />;
                 })}
-            </MuiTabs>
+            </StyledTabs>
             {React.Children.map(children, (child: React.ReactElement<TabProps>, index) => {
+                const ownerState: OwnerState = {
+                    forceRender: child.props.forceRender,
```
Comment (jamesricky) [inline packages/admin/admin/src/tabs/Tabs.tsx:131]: In the case that `index` and `value` are the same, the `display: none` styling was previously not applied, this should stay the same.  

```suggestion
                    contentHidden: index !== value && child.props.forceRender,
```

**Review by jamesricky — CHANGES_REQUESTED** [review]

File: packages/admin/admin/src/tabs/Tabs.tsx (hunk 2 above)
Comment (NataliaVizintini) [reply packages/admin/admin/src/tabs/Tabs.tsx]: should I change `forceRender` to `contentHidden?: boolean` in OwnerState?

Comment (jamesricky) [reply packages/admin/admin/src/tabs/Tabs.tsx]: Yes, as it's not the same value as `props.forceRender` it should at least not be named `forceRender`

**Review by jamesricky — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**